tests/legacy-cli/e2e/tests/build/aot/aot-i18n.ts

```diff
@@ -88,8 +99,16 @@ export default async function () {
   for (const { lang, translation } of langTranslations) {
     if (lang != 'en') {
       await copyFile('src/locale/messages.xlf', `src/locale/messages.${lang}.xlf`);
-      await replaceInFile(`src/locale/messages.${lang}.xlf`, '<source>Hello i18n!</source>',
-        `<source>Hello i18n!</source>\n<target>${translation}</target>`);
+      await replaceInFile(
+        `src/locale/messages.${lang}.xlf`,
+        'source-language="en-US"',
+        `source-language="en-US" target-language="${lang}"`,
```

Comment:
`target-language` was not needed in VE but seems to be required on Ivy.

@clydin looked into this and it seems like the xliff 1.2 parser lists it as optional in http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html.

But `@angular/localize` will throw if it's not there: https://github.com/angular/angular/blob/ec482dadb19eea5cd247eb7428a5d88640bd38f8/packages/localize/src/tools/src/translate/translation_files/translation_parsers/xliff1/xliff1_translation_parser.ts#L56

@petebacondarwin can you shed some light on this? Is this intended, and if so, is there documentation telling users they must do it?

Reply:
Sounds like it's a bug, opened as https://github.com/angular/angular/issues/33323